src/main/java/org/simulator/comp/CompSimulator.java

```diff
@@ -112,7 +112,7 @@ public MultiTable solve(double timeEnd, double stepSize, DESSolver solver)
 
     // Compute the numerical solution of the initial value problem
     // TODO: Rel-Tolerance, Abs-Tolerance.
-    MultiTable solution = solver.solve(interpreter, interpreter.getInitialValues(), 0d, timeEnd);
+    MultiTable solution = solver.solve(interpreter, interpreter.getInitialValues(), 0d, timeEnd, null);
```

Comment:
For this, I will have to make the same copy of the method with just registering the property listener (3-4 lines of code) if any class wants to get the results as they are processed. So, I thought to add null as a parameter to the calls instead of making two methods that are nearly the same. So, should I make a new method with removing the last parameter?

Reply:
> So, should I make a new method with removing the last parameter?
Yes, please add an overloaded method.

Overloading is a common practice. Some benefits:
- The main advantage of this is the cleanliness of code.
- Method overloading increases the readability of the program.
- Overloaded methods give programmers the flexibility to call a similar method for different types of data.